build/files.js

```diff
@@ -122,7 +117,7 @@ const headRegexp = /(^module.exports = \w+;?)/m
     ]
 
     , isBufferReplacement = [
-      /(\w+) instanceof Buffer/
+      /(\w+) instanceof Buffer/g
```

Comment:
Yeah so this one replaces all use of foo instanceof buffer with Buffer.isBuffer(foo)  because the former will not work in the browser but the latter will.  This caused issues with the encoding argument for write and transform